filesystem/path.go

```diff
@@ -56,6 +56,16 @@ func loggedStat(name string) (os.FileInfo, error) {
 	return info, err
 }
 
+// loggedLstat runs os.Lstat (doesn't dereference trailing symlink), but it logs
```

Comment:
This file would then be unnecessary.

Reply:
It's still useful to have `isSymlink()` for the test, so I've kept it.